packages/data/src/settings/resolvers.js

```diff
@@ -3,20 +3,13 @@
 /**
  * Internal dependencies
  */
-import { NAMESPACE } from '../constants';
 import { updateSettingsForGroup, updateErrorForGroup } from './actions';
+import { getAllSettings } from '@woocommerce/wc-admin-settings';
 
-/**
- * External dependencies
- */
-import { apiFetch } from '@wordpress/data-controls';
-
-export function* getSettings( group ) {
-	const url = `${ NAMESPACE }/settings/${ group }`;
+export function* getSetting( group ) {
```

Comment:
So with this change, what this means is if the `getSettings` (note plural) selector is called before `getSetting` selector, it will return an empty object because the state will not have been hydrated yet.  Which is another reason why if you're not querying the server for the data, just eliminate the resolver and hydrate the state during app initialization (without coupling to the server provided data).

Reply:
Another problem with this resolver, is that resolvers must have the same arguments as the sibling selector otherwise resolution state will not be updated correctly.